n2y/converter.py

```diff
@@ -29,6 +29,7 @@
 
 IMAGE_PATH = None
 IMAGE_WEB_PATH = None
+NAMING_SYSTEM = {}
```

Comment:
Is this used only in this module? If so you should follow the naming convention for private variables. See https://peps.python.org/pep-0008/#descriptive-naming-styles